src/components/Navbar/Navbar.jsx

```diff
@@ -32,10 +40,22 @@ export const Navbar = () => {
       <Link to="/">
         <h1 className="brand-name text-m">SK's</h1>
       </Link>
-      <div className="search-box">
-        <input type="search" name="search" placeholder="search ..." />
-        <i className="fas fa-search search-btn"></i>
-      </div>
+      {location.pathname === "/products" ? (
+        <div className="search-box">
+          <input
+            type="search"
+            name="search"
+            placeholder="search ..."
+            onChange={(event) =>
+              filterDispatch({
+                type: SEARCH_FILTER,
+                payload: { search: event.target.value },
```

Comment:
you can call the function here and declare somewhere upper side for batter code readability.